invisible_flow/app.py

```diff
@@ -1,11 +1,33 @@
+from logging import getLogger
+from logging.config import dictConfig
+
 from flask import Flask, render_template, Response, Request
 
 from invisible_flow.globals_factory import GlobalsFactory
 from invisible_flow.storage.storage_factory import StorageFactory
 from invisible_flow.validation import is_valid_file_type
 
+# Logging configuration
+dictConfig({
+    'version': 1,
+    'formatters': {'default': {
```

Comment:
Can we put default on its own line here?  Seems like in most places we keep those keys on their own line